Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -4648,7 +4648,6 @@
 		A1712B4011C7B235007A5315 /* RegExpKey.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = RegExpKey.h; sourceTree = "<group>"; };
 		A18193E11B4E0CDB00FC1029 /* IntlCollatorConstructor.lut.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = IntlCollatorConstructor.lut.h; sourceTree = "<group>"; };
 		A18193E21B4E0CDB00FC1029 /* IntlCollatorPrototype.lut.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = IntlCollatorPrototype.lut.h; sourceTree = "<group>"; };
-		A1B89B88221E002A00EB4CEA /* SDKVariant.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = SDKVariant.xcconfig; sourceTree = "<group>"; };
```

Comment:
I just realized that this patch only removes `SDKVariant.xcconfig` from projects, but doesn't add it in the new location. This won't break the build, but probably isn't what we want?

Reply:
It adds it to `WebKit.xcworkspace` via the `Configurations` folder, as noted in the commit message;

> - Add Configurations folder to WebKit.xcworkspace.

What, specifically, do you propose instead?

Reply:
At one point, I left the `SDKVariant.xcconfig` entries in each Xcode project and turned the local files into symlinks back to the single copy, but that didn't make it obvious that there was a single copy now.

For the Safari workspace, they picked one "Shared" project and added the `xcconfig` files to it in a "SharedConfigurations" pseudo folder, even though the project didn't strictly contain the folder.  We could do that for WebKit, but if so, which project?  WTF?

Reply:
I don't know what exactly adding the folder to workspace achieves. Does it make the file visible in Xcode left panel, somewhere close to other xcconfigs? The commit message just states what was done, not why or what it achieves from user point of view.

What are the pros and cons of what you did vs. a pseudo folder?

Reply:
> What are the pros and cons of what you did vs. a pseudo folder?

AFAICT, you can't create a pseudo-folder outside of an Xcode project in a workspace.  So all comments below are for the read folder currently added to the project.

Pro:  Additional files show up without having to make any further project changes in the WebKit workspace.

Con:  The "other workspace" (which shall not be named per your request) won't show the files because they're not part of a specific Xcode project for WebKit.  (Specifically, if you type in "SDKVariant" in the "other workspace", the WebKit copy of the file won't show up.)

So based on this, we should probably pick one WebKit project to add these files to.  Maybe WTF since it has so much other shared code/headers?

Reply:
> The commit message just states what was done, not why or what it achieves from user point of view.

Well, I had assumed it was obvious that it was added so that people using Xcode to edit files/projects could still find the file.  But this doesn't work for the "other workspace", so it's probably best to add it to one Xcode project—even though the project won't do anything with them—they're just there to expose them in Xcode for viewing/editing.

Reply:
> So based on this, we should probably pick one WebKit project to add these files to. Maybe WTF since it has so much other shared code/headers?

To be clear, I think it's better to use a folder in the WebKit workspace (then add a corresponding folder to the "other" workspace) as it's a bit confusing to arbitrarily put references in the WTF project.